decidim-dev/lib/decidim/dev/test/rspec_support/wicked_pdf_assets_mock.rb

```diff
@@ -8,6 +8,10 @@ module Assets
       def wicked_pdf_stylesheet_pack_tag(*sources)
         stylesheet_pack_tag(*sources)
       end
+
+      # Disables the images in the PDFs as those requests would be jamming under
+      # the test environment
+      def wicked_pdf_image_tag(img, options = {}); end
```

Comment:
Apparently this happens because of the debugging feature in Capybara, so it wants to know always which code was running when an error happened, so it wants to process only a single request at a time due to this.

Therefore, the PDF cannot be generated if generating it would cause more requests when there is already one active request running at the background. Even raising the number of concurrent threads (#9422) does not seem to eliminate this problem.

Reply:
Ok. As far as I see, this makes explicit the behavior that we already had, of images not rendering on PDFs on specs, so I think it's a nice improvement for sure. Thanks for the explanation!